scripts/merge_benchmarks.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+// Vendor
+import fs from 'fs';
+
+// Data
+import desktopAdreno from '../benchmarks/d-adreno.json';
+import desktopAmd from '../benchmarks/d-amd.json';
+import desktopApple from '../benchmarks/d-apple.json';
+import desktopGeforce from '../benchmarks/d-geforce.json';
+import desktopIntel from '../benchmarks/d-intel.json';
+import desktopNvidia from '../benchmarks/d-nvidia.json';
+import desktopRadeon from '../benchmarks/d-radeon.json';
+import mobileAdreno from '../benchmarks/m-adreno.json';
+import mobileApple from '../benchmarks/m-apple.json';
+import mobileIntel from '../benchmarks/m-intel.json';
+import mobileMalit from '../benchmarks/m-mali-t.json';
+import mobileMali from '../benchmarks/m-mali.json';
+import mobilePowerVR from '../benchmarks/m-powervr.json';
+
+const benchmarks = {
```

Comment:
@puckey I want the `merge_benchmarks` were to be merged into `update_benchmarks` but for now this works OK.

Reply:
@gsimone actually had a brilliant suggestion that would work around all of this being necessary: using a CDN like `unpkg` and having the default `benchmarksURL` be `https://unpkg.com/detect-gpu@${pkg_version}/benchmarks` instead of a default `/benchmarks` URL. This would allow everyone to only retrieve the benchmark data required as we had originally intended whilst still giving the user the ability to serve it themselves if they would like. The DRACO library uses a similar technique.

(@puckey)

Reply:
Yes, that sounds like a great solution!